browser/ui/color/BUILD.gn

```diff
@@ -14,3 +14,19 @@ source_set("unit_tests") {
     "//ui/gfx",
   ]
 }
+
+copy("copy_leo_colors") {
```

Comment:
> Out of interest, is this copy necessary? Could we just include the files directly from node_modules?

> Is the point of this to make the import paths nicer?

Yeah that's it. `#include "brave/node_modules/@brave... /colors.h"` . So I would prefer copying the file to the generated dir. In addition, we might need move leo colors to //brave/ui/ in case we need to modify views/ controls. So I think copying it in favor of layering rules in Chromium would be better.